change/@azure-msal-common-7b4fc307-7a4d-40c4-9076-345d5f2c0dad.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Make SHR parameters optional (#3320)",
```

Leaving off the parentheses enables links in the changelog
```suggestion
  "comment": "Make SHR parameters optional #3320",
```